launch/auton.launch

```diff
@@ -5,54 +5,59 @@
                          part of the auton system. Should only be used for testing.
  -->
 <launch>
-    <arg name="run_perception" default="true" />
-    <arg name="sim" default="false" />
-    <arg name="use_ekf" default="true" />
-    <arg name="ekf_start_delay" default="0" />
+    <arg name="run_tag_detector" default="true"/>
+    <arg name="sim" default="false"/>
+    <arg name="use_ekf" default="true"/>
+    <arg name="ekf_start_delay" default="0"/>
 
     <!--
       ==========
       Perception
       ==========
     -->
-    <!-- node to detect AR tags and publish them to the TF tree -->
-    <node if="$(arg run_perception)" name="tag_detector" pkg="mrover" type="tag_detector"/>
+    <node pkg="nodelet" type="nodelet" name="perception_nodelet_manager" respawn="true"
+          args="manager" output="screen"/>
+    <!-- nodelet to detect AR tags and publish them to the TF tree -->
+    <node if="$(arg run_tag_detector)"
+          pkg="nodelet" type="nodelet" name="tag_detector" respawn="true"
+          args="load mrover/TagDetectorNodelet perception_nodelet_manager" output="screen">
+    </node>
 
     <!--
       ===========
       Navigation
       ===========
     -->
```

Comment:
why are you adding this as output? I think it fills up the screen with smach status stuff

Reply:
Oh ok will revert

Reply:
https://answers.ros.org/question/43838/how-can-i-disable-smach-state-transition-log-messages/